### PR TITLE
Retry if a release is not yet available

### PR DESCRIPTION
**Pull request by unguiculus**
Fixes: #69
Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>

### CODE REVIEW

**Review by cpanato — APPROVED** [review]
/lgtm

File: go.mod [hunk 1]
```diff
@@ -10,6 +10,7 @@ require (
 	github.com/google/go-cmp v0.5.2 // indirect
 	github.com/google/go-github/v30 v30.0.0
 	github.com/goreleaser/goreleaser v0.129.0
+	github.com/hashicorp/go-retryablehttp v0.6.7
```
Comment (mattfarina) [inline go.mod:13]: Unfortunately, the MPL-2.0 licensed is not [approved by the CNCF](https://github.com/cncf/foundation/blob/master/allowed-third-party-license-policy.md) and [this repo does not have an exemption](https://github.com/cncf/foundation/blob/master/license-exceptions/cncf-exceptions-2019-11-01.json). We could ask the CNCF for an exemption as they have for other hashicorp repos. Should we ask?

**Review by mattfarina — CHANGES_REQUESTED** [review]
:(

File: go.mod (hunk 1 above)
Comment (cpanato) [reply go.mod:13]: it is being used in other projects: https://github.com/kubernetes/minikube/blob/master/deploy/minikube/release_sanity_test.go#L28

the search query: https://cs.k8s.io/?q=github.com%2Fhashicorp%2Fgo-retryablehttp&i=nope&files=&excludeFiles=&repos=

Comment (unguiculus) [reply go.mod:13]: Not sure if it's worth it. How long would that take? I could just implement the retry myself instead.

Comment (unguiculus) [reply go.mod:13]: @cpanato Interesting. Looks like we'd be in good company.

Comment (unguiculus) [reply go.mod]: I decided to use a different approach for the time being. I figured we already use another retry lib which I am now using.

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**